cli/tests/integration/repl_tests.rs

```diff
@@ -1017,3 +1017,15 @@ fn closed_file_pre_load_does_not_occur() {
       assert_contains!(console.all_output(), "Skipping document preload.",);
     });
 }
+
+#[test]
+fn process_that_fails_to_spawn_prints_its_name_in_err() {
+  util::with_pty(&["repl", "-A"], |mut console| {
+    console.write_line("new Deno.Command('doesntexist').outputSync()");
+    let output = console.read_until("> ");
+    assert_contains!(output, "Failed to spawn: doesntexist");
+    console.write_line("await new Deno.Command('doesntexist').output()");
+    let output = console.read_until("> ");
+    assert_contains!(output, "Failed to spawn: doesntexist");
+  });
+}
```

Comment:
Thanks, but sorry would you be able to rewrite this away from being a repl tests (ex. make it a unit test in tests/unit/command_test.ts instead)? Repl tests take longer to execute so it has a negative impact in the long run if we keep adding repl tests. It would be best to have the repl tests just be for repl specific tests.